rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/QrdaControllerV1.java

```diff
@@ -52,13 +52,18 @@
 	 */
 	@RequestMapping(method = RequestMethod.POST, headers = {"Accept=" + Constants.V1_API_ACCEPT})
 	public ResponseEntity<String> uploadQrdaFile(@RequestParam MultipartFile file) throws IOException {
-		API_LOG.info("Request received " + file.getName());
+		String originalFilename = file.getOriginalFilename();
+		API_LOG.info("Conversion request received for " + originalFilename);
+
 		Converter.ConversionReport conversionReport = qrdaService.convertQrda3ToQpp(
-				new InputStreamSupplierQrdaSource(file.getName(), inputStreamSupplier(file)));
+				new InputStreamSupplierQrdaSource(originalFilename, inputStreamSupplier(file)));
```

Comment:
This will reflect in our DynamoDB audit table now too.  No more boring `file`.

Reply:
excellent!